src/service/resources-impl.js

```diff
@@ -169,6 +173,7 @@ export class Resources {
     });
     this.viewport_.onScroll(() => {
       this.lastScrollTime_ = timer.now();
+      this.framerate_.collect();
```

Comment:
guessing `onscroll` is already throttled? do we need to throttle on the viewer/perf end?